packages/prettier-plugin-java/test/unit-test/lambda/_output.java

```diff
@@ -143,6 +140,49 @@ public void testConstructor() {
       }
     );
   }
+
+  private static <T extends Group> Function<Constructor<?>, T> createInstance(
+    Group entity
+  ) {
+    return ctor ->
+      Try.of(a, () -> {
+        @SuppressWarnings("unchecked")
+        var ng = (T) ctor.newInstance(
+          entity.getId(),
+          entity.getSystemGenerated(),
+          entity.getVersionKey()
+        );
+        return ng;
+      }).getOrElseThrow(ex -> new RuntimeException(ex));
+  }
+
+  void singleLambdaWithBlockLastArgument() {
+    a.of(b, c, d, e -> {
+      return f;
+    });
+  }
```

Comment:
I think there is an issue with formatting this snippet

**Input:**
```java
class T {
    public void method() {
        a.of(
            aaaaaaaaaaaaaaaaaaaaaaaaaa,
            bbbbbbbbbbbbbbbbbbbbbbbbbb,
            cccccccccccccccccccccccccc,
            e -> {
                return f;
            }
        );
    }
}

```
**Output:**
```java
class T {

  public void method() {
    a.of(aaaaaaaaaaaaaaaaaaaaaaaaaa,
    bbbbbbbbbbbbbbbbbbbbbbbbbb,
    cccccccccccccccccccccccccc,
    e -> {
      return f;
    });
  }
}

```

Reply:
Thanks for catching this! I was able to fix this one without much issue. However, it lead me to find another that has me stumped at the moment:
```java
Class T {

  void singleLambdaWithBlockLastArgumentAndLongLambdaArguments() {
    a.of(b,
      (
        cccccccccccccccccccccccccc,
        dddddddddddddddddddddddddd,
        eeeeeeeeeeeeeeeeeeeeeeeeee
      ) -> {
        return f;
      }
    );
  }
}
```
I can't use `ifBreak` to conditionally break before `b,` if the lambda's argument list breaks, because `ifBreak` requires the provided `groupId` to be one that has already been printed (i.e. it must be a `groupId` that appears before the `ifBreak`). The only solution I've thought of so far is grouping the lambda's argument list with the "parent" argument list (in this case, `b`), so that the lambda argument list breaking also breaks the "parent" argument list. However, that seems rather ugly and I'm not even sure if something like that is possible.

Reply:
I was able to resolve both of these. The latter required some additional research, but looking into Prettier's own source code, I realized they were solving this for JS/TS by using `conditionalGroup`, and I was able to do the same.